cliv2/internal/cliv2/cliv2.go

```diff
@@ -360,8 +361,18 @@ func (c *CLI) PrepareV1Command(
 }
 
 func (c *CLI) executeV1Default(proxyInfo *proxy.ProxyInfo, passThroughArgs []string) error {
+	timeout := c.globalConfig.GetInt(configuration.TIMEOUT)
+	var ctx context.Context
+	var cancel context.CancelFunc
+	if timeout == 0 {
+		ctx = context.Background()
+	} else {
+		deadline := time.Now().Add(time.Duration(timeout) * time.Second)
+		ctx, cancel = context.WithDeadline(context.Background(), deadline)
```

Comment:
Suggesting a slightly more concise form,

```suggestion
		ctx, cancel = context.WithTimeout(context.Background(), time.Duration(timeout) * time.Second)
```

Reply:
Why `withTimeout` instead of `withDeadline`?